.envrc

```diff
@@ -1,7 +1,15 @@
 #!/usr/bin/bash
+set -e
```

Comment:
Without it, even when there were errors in the execution of the script it will continue to the end.